src/Forms/Form.tsx

```diff
@@ -0,0 +1,44 @@
+import React, {
+  FunctionComponent,
+  ReactElement,
+  ReactNode,
+  useContext,
+} from 'react';
+import styled, { ThemeContext } from 'styled-components';
+import { fromTheme } from '../Theme';
+
+export interface FormProps {
```

Comment:
I think we may need to include a `name` prop for accessibility reasons

Reply:
The [documentation on `<form>`](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/form) is saying that the `id` should be sufficient - with the `id` and the `aria-label` that was newly added, is this enough?

Reply:
![image](https://user-images.githubusercontent.com/29589689/107659514-3446b080-6c55-11eb-8d1e-bdb002569e5d.png)

Reply:
Ah, you're right. That's my bad.

(I think I was confusing the `name` attribute with the "[accessible name](https://www.w3.org/TR/accname-1.1/#dfn-accessible-name)" concept)